Castanet/Feed.php

```diff
@@ -913,36 +723,52 @@ protected function buildImage(DOMNode $parent)
             $node->appendChild($link);
             $image_node->appendChild($node);
 
-            $width = $document->createTextNode($this->image_width);
+            $width = $document->createTextNode(strval($this->image_width));
```

Comment:
`createTextNode()` expects a string, so I've cast the `int` properties accordingly.